app/services/to_cocina/work/description_mapper.rb

```diff
@@ -62,7 +62,14 @@ def related_resource_params
       end
 
       def access_contact_params
-        CocinaGenerators::Description.contact_emails(contact_emails: work_form.contact_emails_attributes)
+        CocinaGenerators::Description.contact_emails(
+          # Use `#dup` because mutating the passed-in form object would be unexpected
+          contact_emails: work_form.contact_emails_attributes.dup.tap do |contact_emails_attributes|
+            next if work_form.deposits_contact_email.blank?
+
+            contact_emails_attributes << ContactEmailForm.new(email: work_form.deposits_contact_email)
+          end
+        )
```

Comment:
When mapping to Cocina, append the collection contact email to the list of work contact emails.